packages/forma-36-website/src/components/ComponentSource.js

```diff
@@ -3,6 +3,7 @@ import { css } from '@emotion/core';
 import tokens from '@contentful/forma-36-tokens';
 import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import * as f36Components from '@contentful/forma-36-react-components';
+import { Grid, GridItem, Autocomplete } from '@contentful/forma-36-react-components/dist/alpha';
```

Comment:
`import * as f36AlphaComponents from '@contentful/forma-36-react-components';` does not really work for me. Sadly